File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <math.h>

int main(void)
{
    int f = 0;
    int g = 0;
    int h = 0;
    string text = get_string("Text: ");

    int letters = strlen(text);


    //to get number of letters
    for(int z = 0; z < letters; z++)
    {
        if( isalnum(text[z]))
        {
            f++;
        }

    }



    //to get number of words
    for(int x = 0; x < letters; x++)
    {
        if( isspace(text[x]))
        {
            g++;
        }
    }


    //to get number of sentences
    for(int y = 0; y < letters; y++)
    {
        if(text[y] == '.' || text[y] == '!' || text[y] == '?')
        {
            h++;
        }
    }

    //calculation
    float result = (0.0588 * f / (g + 1) * 100) - (0.296 * h / (g + 1) * 100) - 15.8;


    int index = round(result);

    //print results
    if(index  > 16)
    {
        printf("Grade 16+\n");
    }
    else if(index < 1)
    {
        printf("Before Grade 1\n");
    }
    else
    {
        printf("Grade %i\n", index);
    }



}

File: sentimental-readability/readability.py
# TODO

from cs50 import get_string


f = 0
g = 0
h = 0

text = get_string("Text: ")

letters = len(text)

#number of letters
for z in range(letters):
    if (text[z].isalpha()):
        f += 1

#number of words
for x in range(letters):
    if (text[x].isspace()):
        g += 1

#number of sentences
for y in range(letters):
    if (text[y] == '.' or text[y] == '?' or text[y] == '!'):
        h += 1

#CACULATION
coleman = (0.0588 * f / (g + 1) * 100) - (0.296 * h / (g + 1) * 100) - 15.8

index = round(coleman)

if index < 1:
    print("Before Grade 1")
elif index > 16:
    print("Grade 16+")
else:
    print(f"Grade {index}")